packages/Chem/src/descriptors/descriptors-calculation.ts

```diff
@@ -14,12 +15,19 @@ export async function addDescriptors(smilesCol: DG.Column, viewTable: DG.DataFra
   openDescriptorsDialog(await getSelected(), async (selected: any) => {
     await grok.dapi.userDataStorage.postValue(_STORAGE_NAME, _KEY, JSON.stringify(selected));
     selected = await getSelected();
-    const pi = DG.TaskBarProgressIndicator.create('Calculating descriptors');
```

Comment:
Exception must be handled. I believe we need a generic method for this, such as safeCall in the core